nix/t1/run/run-vcs-emu.nix

```diff
@@ -8,26 +8,56 @@ testCase:
 
 assert lib.assertMsg (!emulator.isRuntimeLoad || (dpilib != null)) "dpilib must be set for rtlink emu";
 
-stdenvNoCC.mkDerivation (finalAttr: {
+stdenvNoCC.mkDerivation (rec {
   name = "${testCase.pname}-vcs-result" + (lib.optionalString emulator.enableTrace "-trace");
   nativeBuildInputs = [ zstd jq ];
   __noChroot = true;
 
-  passthru.caseName = testCase.pname;
+  passthru = {
+    caseName = testCase.pname;
+
+    # to open 'profileReport.html' in firefox,
+    # set 'security.fileuri.strict_origin_policy = false' in 'about:config'
+    profile = pkgs.writeShellScriptBin "runSimProfile" ''
+      ${emuDriver} \
+        ${lib.escapeShellArgs emuDriverArgs} \
+        -simprofile time \
+        2> ${testCase.pname}-rtl-event.jsonl
+    '';
+  };
+
+  emuDriver = "${emulator}/bin/${emulator.mainProgram}";
```

Comment:
This will be export to the builder the env, so the following `emuDriver = ${emuDriver}.` can be remove.